style.css

```diff
@@ -1,69 +1,87 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
-  margin: auto;
+  font-family: "Open Sans", sans-serif;
+  padding-bottom: 50px;
 }
 
-.nav-bar .nav-container {
-  background-color: #000000;
-  color: #ffffff;
+.page-bg {
+  background-image: url("https://images.pexels.com/photos/450035/pexels-photo-450035.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260");
+  height: 100vh;
+  width: 100vw;
   display: flex;
-  justify-content: space-between;
   align-items: center;
-  font-family: "Open Sans", sans-serif;
-  padding-left: 0px;
+  justify-content: center;
+  background-size: cover;
+  background-position: center;
 }
-
-.nav-bar a {
-  color: #ffffff;
-  text-decoration: none;
-  padding: 10px 15px 10px 15px;
-  font-weight: normal;
+.page-bg::before {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
 }
-
-.nav-container {
-  padding: 10px 0px;
+.header {
+  z-index: -1;
```

Comment:
This is not needed. You shouldn't use z-index like this. Z-indices should be used carefully.